src/lib/viewers/BaseViewer.js

```diff
@@ -919,8 +918,8 @@ class BaseViewer extends EventEmitter {
     disableAnnotationControls() {
         if (this.annotator && this.annotationControls && this.areNewAnnotationsEnabled()) {
             this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
-            this.annotationControls.resetControls();
             this.annotationControls.toggle(false);
+            this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.RESET));
```

Comment:
Can we move this outside of the conditional above? We'll need it for the react rewrite, which doesn't set `this.annotationControls`.